src/main/cli.js

```diff
@@ -12,8 +11,7 @@ for (const arg of process.argv) {
     global.MARKTEXT_SAFE_MODE = true
     continue
   } else if (arg === '--version') {
-    const gitInfo = global.MARKTEXT_IS_OFFICIAL_RELEASE ? '' : `(${global.MARKTEXT_GIT_INFO})`
-    console.log(`Mark Text: ${app.getVersion()} ${gitInfo}
+    console.log(`Mark Text: ${global.MARKTEXT_VERSION_STRING}
```

Comment:
@fxha How to set `MARKTEXT_VERSION_STRING `?

Reply:
It's automatically set in line 15/16 and is read from the `package.json` `version` field.